tests/typecheck/call.ts

```diff
@@ -28,7 +28,7 @@ const checkCall = async () => {
   call.mute();
   call.reject();
   call.sendDigits('foo');
-  call.sendMessage({ content: 'foo', messageType: Call.MessageType.UserDefinedMessage });
+  call.sendMessage({ content: 'foo', messageType: 'UserDefinedMessage' });
```

Comment:
Let's keep the real value 'user-defined-message' in the test. Later, we'll add more types here.